crates/codegen/parser_templates/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "parser_templates"
```

nit: to match how other crates are defined.

```suggestion
name = "codegen_parser_templates"
```

---

never used, but ok

---

Not blocking, but the standardization makes it easier to grep across all file types.

---

done